Add note to track method for CleverTap (#2227)
segmentio/segment-docs: src/connections/destinations/catalog/clevertap/index.md

When you `track` an event, we will send that event to CleverTap as a custom event.  Note that CleverTap does not support arrays or nested objects for custom track event properties.

> note ""

> > **NOTE:** For your `track` events to appear on CleverTap, an accompanying trait such as `userId` or `email` from `identify` is required in order for CleverTap to record and associate the `track` event.

Reviewer: ```suggestion
> CleverTap requires `identify` traits such as `userId` or `email` to record and associate the Track event. Without these traits, the Track event does not appear in CleverTap.
```
